internal/database/repository/queue.go

```diff
@@ -188,6 +188,9 @@ func (s *PostgresRepository) UpdateQueue(ctx context.Context, queue *dao.Partiti
         allocating_accepted_apps = @allocating_accepted_apps
     WHERE queue_name = @queue_name AND partition = @partition AND deleted_at IS NULL
 `
+	if queue.Partition == "" {
```

Comment:
2 questions:
1. Can you please move the if above the query? It is easier when I read this method to check the invariants, then see the query knowing that all invariants hold
2. In the future, do we implement proper validations on each object like `ValidDAOParititionQueueDAOInfo`? This is the open question and definitely not for this PR, but I just wanted to mention it

Reply:
Seems to be a good idea.